jazz_environment-event-handler/index.js

```diff
@@ -0,0 +1,561 @@
+                                                if (err) {
+                                                    logger.error("ERROR in creating Environment:" + JSON.stringify(err));
+                                                }
+
+                                                logger.debug("Results:" + JSON.stringify(results));
+                                                innerCallback(err, results);
+                                            });
+                                        }
+                                    } else if (
+                                        results.checkInterest.event_name === event_config.UPDATE_ENVIRONMENT ||
+                                        results.checkInterest.event_name === event_config.DELETE_ENVIRONMENT
+                                    ) {
+                                        var updatePayload = {};
+
+                                        svcPayload = {
+                                            uri:
+                                                configData.BASE_API_URL +
+                                                configData.ENVIRONMENT_API_RESOURCE +
+                                                "/" +
+                                                results.getEnvironmentId.environment_id +
+                                                "?domain=" +
+                                                results.populateEnvironmentPayload.service_domain +
+                                                "&service=" +
+                                                results.populateEnvironmentPayload.service_name,
+                                            method: "PUT",
+                                            headers: { Authorization: authToken },
+                                            json: results.populateEnvironmentPayload.environmentPayload,
+                                            rejectUnauthorized: false
+                                        };
+
+                                        logger.debug("Environment API payload for update:" + JSON.stringify(svcPayload));
+
+                                        if (svcPayload !== undefined) {
+                                            requestAPI(svcPayload, function(err, results) {
+                                                if (err) {
+                                                    logger.error("ERROR in updating Environment:" + JSON.stringify(err));
+                                                }
+
+                                                logger.debug("Results:" + JSON.stringify(results));
+                                                innerCallback(err, results);
+                                            });
+                                        }
+									} else if (	results.checkInterest.event_name === event_config.DELETE_BRANCH && 
+												results.populateEnvironmentPayload && 
```

Comment:
fix indentation

Reply:
This seems an issue in the GIT web UI only